examples/compat/ggplot/density.py

```diff
@@ -9,4 +9,4 @@
 
 plt.title("Density ggplot-based plot in Bokeh.")
 
-mpl.to_bokeh(name="density")
+mpl.show(mpl.to_bokeh(name="density"))
```

Comment:
Well, python does do that, so it's not productive really to muse about contrafactuals. Ultimately I imagine the best place for `show`, `save` and `push` to live is in a `bokeh.io` module or something similar, that is available for use by plots created with any API.  My preference would be to hammer away the notion that `show`, `save`, `push` are the functions that work with any API (and are available in any API, yes using magic imports). If, later, we want to deprecate usage from outside `bokeh.io`, OK  we could do that --- but all at once, in a general action across the board, after everything is stable and settled. That's a simple message to convey. 

The alternative is we import `show` from `plotting.py` everywhere for now, but I think that is a confusing message to send. `plotting.py` is already a muddle I don't want to encumber it more.